js/app.js

```diff
@@ -37,23 +246,114 @@ var vm = {
 
 Sammy(function () {
         this.get('#Home', function () {
-            vm.Main.chosenPageId(this.params.page);     
+            vm.Main.chosenPageId(this.params.page);  
             vm.Main.template("page-template")
             vm.Page.name("Home")
-            $.get('./server/home.json',function(data){
-                vm.Page.name(data.title);
-                vm.Page.content(data.content);
-            })
+            // $.get('./server/home.json',function(data){
+            //     vm.Page.name(data.title);
+            //     vm.Page.content(data.content);
+            // })
         });
+//--------------------------------------PLP Page Array---------------------
         this.get('#About', function () {
             vm.Main.chosenPageId(this.params.page);     
-            vm.Main.template("page-template")
+            vm.Main.template("about-template");
+            $.getJSON('http://demo2828034.mockable.io/search/shirts', function(data) {
+                 var unique_prod_arr =[];
+                var Data=[];
+                var clonedArr = data["items"];
+               clonedArr.sort((a,b)=>(a.price <b.price)?-1:((a.price > b.price)? 1 : 0));
+                vm.Product.fisrtRange(clonedArr[0].price);
+                vm.Product.SecondRange(clonedArr[clonedArr.length-1].price);
+                vm.Product.MinVal(clonedArr[0].price);
+                vm.Product.MaxVal(clonedArr[clonedArr.length-1].price);
+            
+                $.each(data["items"], function(i, item) 
+                {
+                    Data.push(item.prodId);
+                });
+
+                var distinct_product =[];
+                $.each(Data, function(i, el){
+                       if($.inArray(el, distinct_product) === -1) distinct_product.push(el);
+                 });
+                    var total_size =[];
+                    var D_total_size=[];
+                    var total_color =[];
+                    var D_total_color=[];
+                    var total_brand =[];
+                    var D_total_brand=[];
+            
+                for(var j=0; j < distinct_product.length; j++)
+                    {
+                        var sample_arr =[];
+                        var C_sample_arr =[];var D_color_arr=[];
```

Comment:
follow same naming convention all the places
variables and function name should in camel case and avoid use of special characters like '_' in the variable name